api/types.go

```diff
@@ -104,3 +104,24 @@ type UserGroup struct {
 	Users            []string `json:"users"`
 	AttributeGroupID string   `json:"attribute_group_id,omitempty"`
 }
+
+// ItAutomation prioritize and remediate routine actions automatically,
+// increase IT efficiency and streamline your processes to reduce performance degrade
+type ItAutomation struct {
```

Comment:
The type name should be `ITAutomation`. Can you please replace that everywhere? (e.g. also in the names of endpoint types and interface methods).